### ISSUE TITLE
reformat push messages to remove url

### ISSUE THREAD

**Issue by ftyers** [apertium/phenny#413]
```
<spectie> apertium-byv-fra: ftyers * apertium-byv-fra.byv-fra.dix: fix a tonne of errors https://github.com/apertium/apertium-byv-fra/commit/84cfc8
<spectie> ->
<spectie> apertium-byv-fra: ftyers * "fix a tonne of errors" [84cfc8]
```

**Comment by jonorthwash** [comment]
Do we really want to remove the url?

**Comment by Androbin** [comment]
The only good reason to do so would be to get rid of the URL preview.
This preview requires special markup on the target.
We might as well rewrite the URL to a proxy.

For example:
https://github.com/apertium/phenny/issues/413 (with markup)
https://proxy.com/github.com/apertium/phenny/issues/413 (without markup)

The proxy would then redirect to the original URL.

**Comment by sushain97** [comment]
> The only good reason to do so would be to get rid of the URL preview.

If I recall correctly, there are settings that could do this. If not, there should be.

> Do we really want to remove the url?

I suggested proxying it via apertium.org. e.g.:

> https://github.com/apertium/apertium-byv-fra/commit/84cfc8

becomes

> https://apertium.org/src/apertium-byv-fra/commit/84cfc8

Which shortens it a bit... but I think it was still too long for @ftyers :)

**Comment by jonorthwash** [comment]
The reason we want a url in the first place is so that it's easy to take a look at what was changed when you see a commit of interest go by.  It doesn't have to be a permalink necessarily.

@ftyers, are you against any URL at all, or do you just think the current ones are too long.  Either way, could you explain your reasoning?

**Comment by ftyers** [comment]
I am not against something short like http://hr.gl/84cfc8 ... anything longer and it's not great. Basically if it makes the typical push report go to two lines instead of one on my screen then it's a bit too long. My preferred option is something like:
```
<begiak> apertium-byv-fra: ftyers * "fix a tonne of errors" [84cfc8]
<spectie> .url 84cfc8
<begiak> https://github.com/apertium/apertium-byv-fra/commit/84cfc8
```

**Comment by jonorthwash** [comment]
I think a short url or an easy way to retrieve a url would be good.  The problem with the latter option is that it's going to be repo-specific, so it's not just "check github and generate url".  On the other hand, begiak could keep a list/database of seen hashes and save the urls as the payloads come in.

This could also be a way to deal with tons of commits (#411).  We could have begiak just say like "20 commits" and list their hashes or something?

**Comment by Androbin** [comment]
How about one of these:
* Provide short-links like `apertium.org/l0r3m1p5um` which show an overview of the commits + links.
* Begiak forwards links to `apertium.org` and `apertium.org/begiak/` lists recent links + search bar.

**Comment by Androbin** [comment]
Fixed in 7d067893ee919bf1cda8386716c5caf69cf89062 with the `!commit` command. (Important: not the `.commit` command)
